addons/test_mail/data/data.xml

```diff
@@ -49,4 +49,40 @@
         <field name="category">default</field>
     </record>
 
+    <record id="rec_comp_test_1" model="res.company">
```

Comment:
Would move to a sepcific file like ``test_alias_mixin_update.xml`` file so that we know it has a specifc usage. With a comment in the file, otherwise people are goign to wonder why it exists and try to remove it :D .